eclipse-theia/theia
# workspace: cleanup multiroot handling

#### What it does


Fixes: https://github.com/eclipse-theia/theia/issues/11533.

The pull-request mainly removes the `workspace.supportMultiRootWorkspace` preference.

The change includes:
- removing `workspace.supportMultiRootWorkspace`.
- removing leftover handling for the preference and simplifying methods.
- adding the `workspace` section of the File` menu for common workspace commands.
- updating `Save Workspace As...` to also work with an unsaved (single folder) workspace and converting it to a saved multiroot workspace.

#### How to test


1. confirm that the application start successfully
2. confirm that single and multi-root workspace are supported (`add folder to workspace...` works well)

#### Review checklist


#### Reminder for reviewers

- As a reviewer, I agree to behave in accordance with [the review guidelines](https://github.com/theia-ide/theia/blob/master/doc/pull-requests.md#reviewing)

Signed-off-by: vince-fugnitto <vincent.fugnitto@ericsson.com>
